docs/index.md

```diff
@@ -78,7 +78,24 @@ OWT Android supports VP8, VP9, H.264 and HEVC video codecs depending on the devi
   + `owt.conference.SubscribeOptions.setVideoOption`
   + `owt.p2p.P2PClientConfiguration.addVideoParameters`
 
-> **Note:** Hardware encoder/decoder requires Android API level 19 or above.
+> **Note:** Hardware encoder/decoder requires Android API level 19 or fabove.
+
+OWT Android supports white video codecs list for extra config file in /sdcard/mediaCodec.xml:
```

Comment:
OWT AndroidSDK supports adding white-listed video codecs in extra config file in /sdcard/mediaCodec.xml; A sample config file is shown below:

Reply:
done